docs/components/constants.js

```diff
@@ -15,29 +15,19 @@ const whiteSpace = system({
   }
 })
```

Comment:
These lint changes are here because I went ahead and updated eslint-github to `3.4.0` :)